doc/99200-WinCEPlatformNotes.page

```diff
@@ -3,130 +3,7 @@ AAAIntroduction
-Timezone_WINCE.cpp and ThreadPool.cpp.
-
-To obtain the current date and time with millisecond resolution,
-the hack described in <http://community.opennetcf.com/articles/cf/archive/2007/11/20/getting-a-millisecond-resolution-datetime-under-windows-ce.aspx>
-is used. This means there will be a one second delay when starting up
-the application.
-
-
-!!Poco::Path
-
-Poco::Path::listRoots() returns the root directory ("\"), as well as all mounted storage devices
-(e.g., "\Hard Disk"), even if they are also present in the root directory.
-
-Poco::Path::current() and Poco::Path::home() always return the root directory.
-
-
-!!Poco::RWLock
-
-In the reader/writer lock implementation for Windows CE, writers always take precedence over readers.
-
-
-!!Poco::Process
-
-Launching a process with pipes for redirecting input/output is not supported.
-
-
-!!Poco::Util::ServerApplication
-
-Poco::Util::ServerApplication::waitForTerminationRequest(): CTRL-C does not work to shut down the application as
-it's not supported by the Windows CE console. The pkill utility supplied as part of the Util library
-samples can be used to terminate a process from the command line.
-
-
-!!Crypto and NetSSL
-
-Crypto and NetSSL_OpenSSL have not been tested yet. Project files are there, but they might need some adaptions depending on
-how OpenSSL has been built for the Windows CE target.
-
-
-!!Data
-
-Only the SQLite backend is currently supported.
-
-The SQLite engine in Data/SQLite is built without localtime support (SQLITE_OMIT_LOCALTIME) due to localtime_s() not being
-available on Windows CE.
-
-
-!!Raw Sockets in Net
-
-The test cases involving raw sockets will fail unless the testsuite is ran as a privileged (signed) application.
-These are RawSocketTest::testEchoIPv4(), RawSocketTest::testSendToReceiveFromIPv4() and ICMPClientTest::testPing().
-
-
-!!!Build Notes
-
-Optimization settings should be set as follows for release builds
-(<*Properties > Configuration Properties > C/C++ > Optimization*>):
-  * Optimization: Custom
-  * Inline Function Expansion: Default
-  * Enable Intrinsic Functions: Yes (/Oi)
-  * Floating-Point Consistency: Default Consistency
-  * Favor Size or Speed: Favor Fast Code (/Ot)
-  * Whole Program Optimization: No
-
-Other settings may or may not produce working programs.
-Specifically, setting <*Optimization*> to <*Maximize Speed (/O2)*> will result in failures in the
-test suite for Foundation Events due to a compiler optimizer bug.
-
-For shared/DLL builds, the /FORCE:MULTIPLE option is passed to the linker. This is
-to avoid a problem with iostream classes and their methods (template instantiations),
-which for some unknown reason (possibly bug) will be exported by the Foundation library
-(and others) and thus cause multiply defined symbol errors.
-
-The reference system used for testing is a Digi ConnectCore 9P 9360 running Windows CE 6.0.
+Starting with release 1.4.0 the POCO C++ Libraries cannot be used on Windows CE
```

Comment:
Need to update version number to something else. Will do this at some point if removing this get's green light and we know if this going to land 1.13.0 or 1.14.0.